src/test/java/org/rulez/demokracia/pdengine/ObtainBallotTest.java

```diff
@@ -75,6 +75,27 @@ public void obtainBallot_stores_the_ballot() {
 		assertTrue(vote.ballots.contains(ballot));
 	}
 
+	@tested_feature("Manage votes")
+	@tested_operation("Obtain ballot")
+	@tested_behaviour("the number of ballots obtained with adminKey are recorded for \"admin\"")
+	@Test
+	public void obtainBallot_increases_recordedBallots_when_adminKey_is_admin() {
+		String voteId = adminInfo.voteId;
+		Vote vote = voteManager.getVote(voteId);
+		int originalObtainedBallots = vote.getRecordedBallots("admin");
```

Comment:
this is not "admin", but vote.adminKey

Reply:
Sorry, It was a long, day and I dont get it what you would like to change. : ( :)

Reply:
s/"admin"/vote.adminKey/

Reply:
No one is rushing you, take your time.
If you want to play safe, you can ask for more time in this pull request, this is not in critical path.
But we are not yet actively enforcing the expiry rule anyways. You can find even a 5 days old issue in working.

Reply:
Fixed :)